### PR TITLE
[THEMES] New ReactOS theme - Lunar

### PR DESCRIPTION
**Pull request by archeYR**
Lunar is a theme made specifically for ReactOS. We decided to go with a mix of ReactOS colors and some XP-like tones.
Proposed theme is based off Lautus, artwork is done by @Cernodile while I(@archeYR ) was doing theme adjustements for the new assets.

Currently missing:
Start menu, will be introduced in a later PR as we do not have modern start menu right now on ReactOS.

### CODE REVIEW

**Comment by binarymaster** [comment]
Care to attach some screenshots to PR? :slightly_smiling_face:

**Comment by archeYR** [comment]
VirtualBox_ReactOS2_20_03_2019_22_56_58
VirtualBox_ReactOS2_20_03_2019_22_26_56
VirtualBox_ReactOS2_20_03_2019_22_26_06

File: sdk/cmake/CMakeMacros.cmake [hunk 1]
```diff
@@ -296,6 +296,8 @@ macro(dir_to_num dir var)
         set(${var} 62)
     elseif(${dir} STREQUAL reactos/3rdParty)
         set(${var} 63)
+	elseif(${dir} STREQUAL reactos/Resources/Themes/Lunar)
```
Comment (binarymaster) [inline sdk/cmake/CMakeMacros.cmake]: Indentation is broken here.

File: media/themes/lunar.msstyles/lang/en-US.rc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{
+5000, "Lunar"
+5001, "Visual Style"
+5002, "ReactOS Foundation"
+5003, "Cernodile & Illen, built on top of Lautus"
+5004, "Released under GNU/GPL 2.0, 2019"
+5005, "http://www.reactos.org"
+5006, "1.0"
+5007, "Visual Style for ReactOS"
+}
```
Comment (binarymaster) [inline media/themes/lunar.msstyles/lang/en-US.rc]: No newline at end of file.

File: media/themes/CMakeLists.txt [hunk 3]
```diff
@@ -2,3 +2,4 @@
 add_subdirectory(blackshade.msstyles)
 add_subdirectory(lautus.msstyles)
 add_subdirectory(modern.msstyles)
+add_subdirectory(lunar.msstyles)
```
Comment (binarymaster) [inline media/themes/CMakeLists.txt]: No newline at end of file.

Comment (ThFabba) [inline media/themes/CMakeLists.txt]: Keep alphabet order please

**Comment by p01arst0rm** [comment]
nicely done, worthy addition

**Comment by yagoulas** [comment]
Finallly a theme for ros that isn't an abomination!

**Comment by p01arst0rm** [comment]
Can this be merged soon? i want to submit another PR but it depends on this being accepted

**Pull request merged**